### PR TITLE
Feature/arinaho/quiz complete

### PR DESCRIPTION
**Pull request by arinaho-muleba**
Screenshot 2024-05-06 214221
Screenshot 2024-05-06 214335

### CODE REVIEW

**Comment by tebogo-bbd** [comment]
Looking great. My problem is with the past attempts button the text is not readable. Maybe making the text color the same color as the play again button

File: frontend/assets/images/meme.png
Comment (tebogo-bbd) [inline frontend/assets/images/meme.png:1]: move this to a memes folder I'm planning on adding a meme for different scores a user can get

**Review by tebogo-bbd — CHANGES_REQUESTED** [review]

File: frontend/assets/images/meme.png
Comment (arinaho-muleba) [reply frontend/assets/images/meme.png:1]: sure

**Review by tebogo-bbd — APPROVED** [review]

**Review by sudosf — CHANGES_REQUESTED** [review]
Please update the general buttons styles to use their own class names

File: frontend/css/style.css [hunk 1]
```diff
@@ -115,14 +123,14 @@ button:hover,
 
 main,
 header {
+    padding: 0.5rem;
     display: flex;
-    margin: 2.5rem;
+    /* margin: 2.5rem; */
```
Comment (sudosf) [inline frontend/css/style.css]: bring back the margin

**Review by sudosf — APPROVED** [review]
remove commented padding and change `flex: 1 100%` to just `flex: 1`

**Pull request merged**